src/renderer/gl_engine/tvgGlTessellator.cpp

```diff
@@ -1655,38 +1655,42 @@ void Stroker::doStroke(const PathCommand *cmds, uint32_t cmd_count, const Point
     mResGlPoints->reserve(pts_count * 4 + 16);
     mResIndices->reserve(pts_count * 3);
 
+    auto hasLineTo = false;
```

Comment:
this doesn't mean only lineTo by curveTo. Suggestion:
```
auto validStrokeCap 
```

Reply:
@hermet Done.